test/controllers/observer_controller_test.rb

```diff
@@ -1350,6 +1350,19 @@ def test_destroy_observation
     end
   end
 
+  def test_destroy_observation_failure
+    obs = observations(:minimal_unknown_obs)
+    params = { id: obs.id.to_s }
+    login(obs.user.login)
+
+    Observation.any_instance.stubs(:destroy).returns(false)
```

Comment:
**Question: Is there a way to make `obs.destroy` return `false` without stubbing?**
For instance, is there something I can do to `obs` so that `destroy` blows up?
The object here is to cover these lines in `CreateAndEditObservation`:
```ruby
426     elsif !@observation.destroy
427       flash_error(:runtime_destroy_observation_failed.t(id: obs_id))
428       redirect_to(add_query_param({ action: "show_observation", id: obs_id },
429                                   this_state))
```

**Question: If stubbing is best, is there a better way to stub `obs.destroy`?**
I tried `obs.stub(:destroy, false) do ...` but it didn't work. Do I have the syntax wrong?